src/Symfony/Component/Routing/Tests/Loader/ObjectRouteLoaderTest.php

```diff
@@ -13,140 +13,19 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Routing\Loader\ObjectRouteLoader;
-use Symfony\Component\Routing\Route;
-use Symfony\Component\Routing\RouteCollection;
 
 class ObjectRouteLoaderTest extends TestCase
```

Comment:
please revert changes on this test case and just mark it as `@group legacy`, then copy-paste tests that are relevant to the new class's test case